packages/themes/src/sportsbetDark/sportsbetDark.ts

```diff
@@ -9,38 +9,44 @@ const boxShadow =
 
 const color = {
   piccolo: {
-    120: '#02c054',
-    100: '#0ccf61',
-    80: '#18dc6d',
+    120: '#49B356',
+    100: '#49B356',
+    80: '#62BE6D',
   },
   hit: {
-    120: '#2d3139',
-    100: '#393e47',
-    80: '#454a54',
+    120: '#4C515A',
+    100: '#4C515A',
+    80: '#4C515A',
   },
   beerus: {
-    100: '#31373f',
+    100: '#FFFFFF1F',
```

Comment:
are you sure that this is correct ccolor

Reply:
Yes, it is. As per design.
![image](https://user-images.githubusercontent.com/30629322/203038321-f6cfac5b-7367-447f-a33f-a12e7deece1f.png)